pumpkin/src/command/commands/cmd_pumpkin.rs

```diff
@@ -59,6 +59,16 @@ impl CommandExecutor for PumpkinExecutor {
                             "Click to Copy Minecraft Version",
                         )))
                         .color_named(NamedColor::Gold),
+                    )
+                    .add_child(
+                        TextComponent::text("Github Repository")
```

Comment:
Github -> GitHub

Maybe also add the doc url: https://snowiiii.github.io/Pumpkin/
like **GitHub**, **Documentation**

Reply:
ok

Reply:
No like as a text you can click

Reply:
how?

Reply:
Another TextComponent

Reply:
oh i didint understand added